_gsocproposals/2020/proposal_ATLASMLcompression.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Deep autoencoders for ATLAS data compression
+layout: gsoc_proposal
+project: ATLAS
+year: 2020
+organization: CERN
+---
+
+## Short description of the project 
+
+Storage is one of the main limiting factors to the recording of information from proton-proton collision events at the Large Hadron Collider, at CERN in Geneva. Hence, ATLAS uses a so-called ”trigger system”, which selects and sends interesting events to the data storage system while throwing away the rest. 
```

Comment:
Please add inline links on LHC, CERN and possibly "trigger system"

Reply:
Fixed in Master